jobs/ads-incrementality-dap-collector/tests/test_mock_responses.py

```diff
@@ -48,20 +51,23 @@
             "ratio": 1,
             "features": [
                 {
-                    "featureId": "dapTelemetry",
+                    "featureId": "dapIncrementality",
                     "enabled": True,
                     "value": {
-                        "enabled": True,
-                        "visitCountingEnabled": True,
-                        "visitCountingExperimentList": [
+                        "advertiser": "Bookshop",
+                        "taskId": "0QqFBHvuEk1_y4v4GIa9bTaa3vXXtLjsK64QeifzHpo",
+                        "length": 4,
+                        "timePrecision": 3600,
+                        "measurementType": "referrerMeasurement",
+                        "referrerUrls": [
                             {
-                                "name": "1841986",
+                                "url": "*://*.bookshop.com/",
                                 "bucket": 2,
-                                "task_id": "mubArkO3So8Co1X98CBo62-lSCM4tB-NZPOUGJ83N1o",
-                                "task_veclen": 4,
-                                "urls": ["*://*.glamazon.com/"],
+                                "metric_name": "organic_conversions_tile_pinned",
                             }
                         ],
+                        "targetUrls": "*://*.bookshop.com/thankyou.html?*orderId*",
```

Comment:
Update the `targetUrls` to be more generic, e.g `*://*.bookshop.com/page.html`.  This applies to all `targetUrls`.

Reply:
I did this for the target urls -- what do you think about doing the same for referrer urls? Should I leave a generic query param in there, or should we go for the same `page.html` suffix there?